pkgs/servers/sql/postgresql/ext/apache_datasketches.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv, lib, fetchFromGitHub, gcc, postgresql, boost182, nixosTests }:
+let
+  cpp_sources = {version, hash}: fetchFromGitHub {
+    owner  = "apache";
+    repo   = "datasketches-cpp";
+    rev = "refs/tags/${version}";
+    sha256 = hash;
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache_datasketches";
+  version = "1.6.0";
+  hash = "sha256-sz94fIe7nyWhjiw8FAm6ZzVpB0sAK5YxUrtbaZt/guA=";
+
+  cpp_version = "4.1.0";
+  cpp_hash = "sha256-vPoFzRxOXlEAiiHH9M5S6255ahzaKsGNYS0cdHwrRYw=";
+
+  src = fetchFromGitHub {
+    owner  = "apache";
+    repo   = "datasketches-postgresql";
+    rev    = "refs/tags/${finalAttrs.version}";
+    sha256 = finalAttrs.hash;
```

Comment:
1. There's no need to create an attribute for this, please add the hash as a string inside `fetchFromGitHub`.
2. The attribute `sha256` should be renamed into `hash`.

Reply:
How does this work when the updated version results in a download with new hash?

Reply:
I don't understand.

When there's a new version, the version and the hash needs to be updated. Both of them.

Reply:
If i understood the purpose of finalAtts, we're leaving the version as overridable by the user - but someone just sending in version=1.6.1 will result in a download with a different hash. Or did I misunderstand the purpose of finalAttrs?

Reply:
They will have to send also a `src` attribute then in that case.